web/src/user_status_ui.js

```diff
@@ -51,7 +51,7 @@ export function submit_new_status() {
     let old_status_text = user_status.get_status_text(user_id) ?? "";
     old_status_text = old_status_text.trim();
     const old_emoji_info = user_status.get_status_emoji(user_id) || {};
-    const new_status_text = input_field().val().trim();
+    const new_status_text = input_field().val().toString().trim();
 
```

Comment:
Don’t do this. The correct way to make TypeScript understand that `.val()` returns a string is to convince it that the input is a `JQuery<HTMLInputElement>`, not `JQuery<HTMLElement>`.

Reply:
Adding a commit to correct it soon. Thanks!